docs/Onboarding/Onboarding_Guide.md

```diff
@@ -0,0 +1,47 @@
+# OpenShift CI Scenario Onboarding Guide<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [Overview](#overview)
+- [Purpose](#purpose)
+- [Onboarding Workflow Diagram](#onboarding-workflow-diagram)
+- [Onboarding Phases](#onboarding-phases)
+  - [1. Prerequisites](#1-prerequisites)
+  - [2. Scenario Kick-off](#2-scenario-kick-off)
+  - [3. Develop Scenario](#3-develop-scenario)
+  - [4. Maintenance \& Expansion](#4-maintenance--expansion)
+  - [5. Celebrate \& Share](#5-celebrate--share)
+
+## Overview
+This Onboarding guide is meant to describe a repeatable process that can be followed to create test scenarios for OpenShift integrated products using OpenShift CI. The goal is that we can onboard & expand new scenarios that are easy to understand, maintain and debug.
+
+We must deeply understand the need for each specific scenario that we put through this process and update this process when needed. This is not meant to be followed blindly so that we can onboard scenarios faster. It is meant to teach a generic way to onboard scenarios that is proven to work. Its expected that new scenarios will present new problems and we must develop better solutions that what is proposed. We need to hear your painpoints and feedback throughout every step of this process. **Please communicate over the [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) slack channel**
```

Comment:
```suggestion
We must deeply understand the need for each specific scenario that we put through this process and update this process when needed. This is not meant to be followed blindly so that we can onboard scenarios faster. It is meant to teach a generic way to onboard scenarios that is proven to work. Its expected that new scenarios will present new problems and we must develop better solutions than what may be proposed in this document. We need to hear your painpoints and feedback throughout every step of this process. 
> **NOTE: **
> Please communicate over the [#forum-qe-cspi-ocp-ci](https://coreos.slack.com/archives/C047Y0DPEJU) Slack channel
```

Reply:
fixing